src/components/time-picker/time-picker.html

```diff
@@ -0,0 +1,21 @@
+<div class="bx--form-item">
+  <label for="time-picker-1" class="bx--label">Label</label>
+  <div id="time-picker-1" data-time-picker class="bx--time-picker">
+    <div data-time-picker-input class="bx--time-picker__input">
+      <input data-time-picker-input-field type="number" class="bx--time-picker__input-field bx--time-picker__input-field--hours" placeholder="hh" step="1" min="0" />
+      <span class="bx--time-picker__separator">:</span>
+      <input data-time-picker-input-field type="number" class="bx--time-picker__input-field bx--time-picker__input-field--mins" placeholder="mm" step="1" min="0" />
+      <input data-time-picker-input-field type="text" class="bx--time-picker__input-field bx--time-picker__input-field--text" pattern="[a-zA-Z]{2,2}" placeholder="am" maxlength="2" />
+    </div>
+    <div class="bx--time-picker__select bx--select bx--select--inline">
+      <select id="select-id" class="bx--select-input">
+        <option class="bx--select-option" value="CST">CST</option>
```

Comment:
Did you really want time zone selection box in the example HTML? I'd bet the time zone calculation logic, which requires heavier library, would be on us (instead of application does it with the library of their choice) as soon as we add this UI.